### PR TITLE
kabir/wc-204

### PR DESCRIPTION
**Pull request by kabirsikand**

### CODE REVIEW

File: content/workers/platform/pricing.md [hunk 1]
```diff
@@ -38,26 +38,28 @@ You may change the Usage Model for individual Workers without affecting your acc
 
 ### Same features
 
-Script size, number of scripts, subrequests, and available memory are not affected by plan type.
+See [Limits](/workers/platform/limits/). Script size, number of scripts, and available memory are not affected by plan type. The Unbound usage model includes a higher number of subrequests.
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
Script size, number of scripts, and available memory are not affected by plan type. The Unbound usage model includes a higher number of subrequests. Refer to [Limits](/workers/platform/limits/). 
```

Comment (deadlypants1973) [reply content/workers/platform/pricing.md]: Are we still using "script" vocabulary here? I thought we were phasing it out?

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -217,6 +223,12 @@ Once a Worker has six connections open, it can still attempt to open additional
 
 If the system detects that a Worker is deadlocked on open connections — for example, if the Worker has pending connection attempts but has no in-progress reads or writes on the connections that it already has open — then the least-recently-used open connection will be canceled to unblock the Worker. If the Worker later attempts to use a canceled connection, an exception will be thrown. These exceptions should rarely occur in practice, though, since it is uncommon for a Worker to open a connection that it does not have an immediate use for.
 
+{{<Aside type="note">}}
+
+Simultaneous Open Connections are measured from the top-level request, meaning any connections open from Workers sharing resources (e.g. Workers triggered via [Service Bindings](/workers/runtime-apis/service-bindings/)) will share the simultaneous open connection limit.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
Simultaneous Open Connections are measured from the top-level request, meaning any connections open from Workers sharing resources (for example, Workers triggered via [Service bindings](/workers/runtime-apis/service-bindings/)) will share the simultaneous open connection limit.
```

File: content/workers/platform/limits.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ title: Limits
 
 | Feature                                                                         | Free      | Paid      |
 | ------------------------------------------------------------------------------- | --------- | --------- |
-| [Subrequests](#subrequests)                                                     | 50        | 50        |
+| [Subrequests](#subrequests)                                                     | 50        | 50 Bundled, 1000 Unbound        |
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md:14]: If everything is the same for Bundled and Unbound under Paid, then we should change "Paid" to > "Bundled, Unbound" to be clearer.

File: content/workers/platform/limits.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ title: Limits
...
 | [Simultaneous outgoing<br/>connections/request](#simultaneous-open-connections) | 6         | 6         |
 | [Environment variables](#environment-variables)                                 | 64/worker | 64/worker |
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
| [Environment variables](#environment-variables)                                 | 64/Worker | 64/Worker |
```

File: content/workers/platform/limits.md [hunk 5]
```diff
@@ -169,7 +169,13 @@ Use the [TransformStream API](/workers/runtime-apis/streams/transformstream/) to
 
 ## CPU runtime
 
-Most Workers requests consume less than a millisecond. It is rare to find a normally operating Workers script that exceeds the CPU time limit. A Worker may consume up to 10 ms on the Free plan and up to 50 ms for Bundled Workers on the Paid Plan. The Paid Plan also offers up to a 30 second [duration](/workers/platform/limits/#duration) for increased compute time. The 10 ms allowance on the Free plan is enough execution time for most use cases including application hosting.
+Most Workers requests consume less than a millisecond. It is rare to find a normally operating Workers script that exceeds the CPU time limit. CPU time is capped at various limits depending on your plan, usage model, and Worker type. 
+
+* A Worker may consume up to **10 ms** on the Free plan.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
* A Worker may consume up to **10 milliseconds** on the Free plan.
```

File: content/workers/platform/limits.md [hunk 6]
```diff
@@ -169,7 +169,13 @@ Use the [TransformStream API](/workers/runtime-apis/streams/transformstream/) to
...
+* A Worker or [Scheduled Worker](/workers/platform/cron-triggers/) may consume up to **50 ms** with the Bundled usage model on the Paid Plan.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
* A Worker or [Scheduled Worker](/workers/platform/cron-triggers/) may consume up to **50 milliseconds** with the Bundled usage model on the Paid Plan.
```

**Review by Albertozhao — APPROVED** [review]

**Pull request merged**